includes/Shopping/Shopify_Query.php

```diff
@@ -242,17 +247,21 @@ protected function fetch_remote_products( string $search_term ) {
 		return $result;
 	}
 
+	
 	/**
 	 * Get products by search term.
 	 *
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term.
+	 * @param string $orderby sort field for query.
+	 * @param string $order ASC or DESC.
```

Comment:
Nit: It's lowercase asc and desc since that's what `get_products_query` uses for comparison.

```suggestion
	 * @param string $orderby Sort retrieved products by parameter. Default 'date'.
	 * @param string $order   Whether to order products in ascending or descending order.
     *                        Accepts 'asc' (ascending) or 'desc' (descending). Default 'desc'.
```